### PR TITLE
Fix / initialized user stream in huobi

### PR DESCRIPTION
**Pull request by vic-en**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

### CODE REVIEW

**Comment by RC-13** [comment]
Confirmed and tested by QA, delay no longer persist. Test done locally and on the cloud, with 30 sec refresh rate, after orders are successfully cancelled, new one will be created immediately. 
huobi

**Review by RC-13 — APPROVED** [review]

**Comment by rxlxrxsx** [comment]
We're seeing that the bot freezes under this branch when running from some time, this doesn't happen when running on `master` or `development` branch. I checked the logs but there were no signs of network interruption or errors during the time the bot freezes. Here are the logs in case you want to check.

[logs_huobi-test.log](https://github.com/CoinAlpha/hummingbot/files/5833622/logs_huobi-test.log)
[logs_huobi_ht-usdt-test01.log](https://github.com/CoinAlpha/hummingbot/files/5833623/logs_huobi_ht-usdt-test01.log)

**Comment by rapcmia** [comment]
Checked this as well locally and no delays when order is cancelled/created
image

**Comment by RC-13** [comment]
Test bot has been up and running now for 2 days and more than 9 hours and never freeze

File: hummingbot/connector/exchange/huobi/huobi_exchange.pyx [hunk 1]
```diff
@@ -599,8 +613,10 @@ cdef class HuobiExchange(ExchangeBase):
                         continue
 
                     execute_amount_diff = s_decimal_0
-                    execute_price = Decimal(data["tradePrice"])
-                    remaining_amount = Decimal(data["remainAmt"])
+                    # tradePrice is documented by Huobi but not sent.
+                    # However, orderprice isn't applicable to all order_types, so we assume tradePrice will be sent for such orders
+                    execute_price = Decimal(data.get("orderPrice", data.get("tradePrice", "0")))
+                    remaining_amount = Decimal(data.get("remainAmt", data["orderSize"]))
```
Comment (CrimsonJacket) [inline hummingbot/connector/exchange/huobi/huobi_exchange.pyx:619]: Should we assume there is `tradePrice` first before `orderPrice`? From past interactions, I do remember `tradePrice` being more accurate than `orderPrice`. Also I am assuming that there wouldn't be a case where `execute_price` would be `Decimal("0")` right?

Comment (CrimsonJacket) [reply hummingbot/connector/exchange/huobi/huobi_exchange.pyx:619]: After some discussion wth @vic-en, using `order_price` to reflect the `execute_price` will suffice in for `LIMIT` order types.

**Review by CrimsonJacket — APPROVED** [review]
LGTM 👍

**Pull request merged**